### DISCUSSION TITLE
Unable to toggle Enable National Vulnerability Database mirroring

### DISCUSSION THREAD

**Discussion by frost9i** [DependencyTrack/dependency-track#1545] [Q&A]
I seem to be unable to enable Enable National Vulnerability Database mirroring after it was disabled.
As well as I cannot re-add NVD feed URL after the original one was changed -> https://nvd.nist.gov/feeds

![image](https://user-images.githubusercontent.com/25299818/163139379-b130f723-024a-4f40-afd0-47b9d64b08d8.png)

Can someone please check this behavior? Should I create an issue?

**Comment by stevespringett** [comment]
Likely due to https://github.com/DependencyTrack/frontend/issues/137
